{"name": "test_multiple_login_page_success[invalid_user3-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user3'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753502041736, "stop": 1753502042661, "uuid": "40fe05d1-59f4-4ba1-82d4-55f9ad02305b", "historyId": "3aefb0a77e91fb9168e6dfda7f61ecfa", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "52384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}